gdal/frmts/jpegxr/frmt_jpegxr.html

```diff
@@ -0,0 +1,80 @@
+<html>
+<head>
+<title>JPEGXR -- JPEG XR File Format</title>
+</head>
+
+<body bgcolor="#ffffff">
+
+<h1>JPEGXR -- JPEG XR File Format</h1>
+
+<p>The JPEG XR format is supported for reading, and batch writing, but not
+update in place.  JPEG XR files are represented as one band (Greyscale) or three
+band (RGB) or four band (RGB with Alpha) datasets with Byte valued bands.<p>
+
+<p>The driver is implemented based on the 
+    <a href="https://jxrlib.codeplex.com/">jxrlib</a> - open source (BSD)
+    implementation of JPEG XR format by Microsoft.</p>
```

Comment:
@schwehr It's just what my text editor does (wrap & indent). Since, GDAL's docs seem(ed) to be pretty free-form HTML, I didn't reformat it. What is the desired format?